### PR TITLE
Median filter

### PR DESCRIPTION
**Pull request by LijieTu**
Add a median filter

### CODE REVIEW

File: tomviz/AddPythonTransformReaction.cxx [hunk 1]
```diff
@@ -449,6 +449,43 @@ OperatorPython* AddPythonTransformReaction::addExpression(DataSource* source)
...
+  {
+     QDialog dialog(pqCoreUtilities::mainWidget());
+     dialog.setWindowTitle("Apply Median filter");
+     QGridLayout *layout = new QGridLayout;
+     QLabel *labelDescription = new QLabel("Apply an isotropic Median filter. \nThe size can be specidied below:");
+     layout->addWidget(labelDescription,0,0,1,2);
+    
+     QLabel *label = new QLabel("Size:", &dialog);
+     layout->addWidget(label);
+     QDoubleSpinBox *size = new QDoubleSpinBox(&dialog);
```
Comment (yijiang1) [inline tomviz/AddPythonTransformReaction.cxx]: A QSpinBox is sufficient because the argument is an integer.

**Comment by LijieTu** [comment]
Changed doubleSpin to Spin

**Comment by Hovden** [comment]
@yijiang1

**Comment by yijiang1** [comment]
Works well (OSX).

**Pull request merged**